### DISCUSSION TITLE
Still reverts on unclaim

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6479] [Q&A] [answered]
So, for some reason one of my players had there base deleted. Im assuming they unclaimed it but revert on unclaim false? Any help.

![image](https://user-images.githubusercontent.com/116373326/217565640-3eb15a8f-9459-4495-9b80-c5312fa648cd.png)

**Comment by LlmDl** [comment, marked as answer]
FAQ question #2: https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions
